Sdiff -u -r -x .DS_Store src/tap/Info.plist src_/tap/Info.plist
--- src/tap/Info.plist	2014-08-10 16:47:35.000000000 -0400
+++ src_/tap/Info.plist	2015-10-26 09:45:09.000000000 -0400
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
@@ -15,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>@@CFBUNDLEPACKAGETYPE@@</string>
 	<key>CFBundleShortVersionString</key>
-	<string>@@CFBUNDLEVERSION@@</string>
+	<string>@@CFBUNDLEVERSION@@ (Tunnelblick tap kext version TBKEXTBUILDNUMBER; Tunnelblick build TBBUILDNUMBER)</string>
 	<key>CFBundleSignature</key>
 	<string>@@CFBUNDLESIGNATURE@@</string>
 	<key>CFBundleVersion</key>
-	<string>1.0</string>
+	<string>TBKEXTBUILDNUMBER</string>
 	<key>OSBundleLibraries</key>
 	<dict>
 		<key>com.apple.kpi.mach</key>
@@ -33,4 +33,3 @@
 	</dict>
 </dict>
 </plist>
-
--- src/tun/Info.plist	2014-08-10 16:47:35.000000000 -0400
+++ src_/tun/Info.plist	2015-10-26 09:45:50.000000000 -0400
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
@@ -15,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>@@CFBUNDLEPACKAGETYPE@@</string>
 	<key>CFBundleShortVersionString</key>
-	<string>@@CFBUNDLEVERSION@@</string>
+	<string>@@CFBUNDLEVERSION@@ (Tunnelblick tun kext version TBKEXTBUILDNUMBER; Tunnelblick build TBBUILDNUMBER)</string>
 	<key>CFBundleSignature</key>
 	<string>@@CFBUNDLESIGNATURE@@</string>
 	<key>CFBundleVersion</key>
-	<string>1.0</string>
+	<string>TBKEXTBUILDNUMBER</string>
 	<key>OSBundleLibraries</key>
 	<dict>
 		<key>com.apple.kpi.mach</key>
@@ -33,4 +33,3 @@
 	</dict>
 </dict>
 </plist>
-
